examples/Avatar.main.tsx

```diff
@@ -36,7 +36,7 @@ export default () => {
       <Avatar
         abbreviation='RR'
         backgroundColor={getUserColor('Robbie Robot')}
-        image={<img src={imgSrc} />}
+        image={<img src={imgSrc} alt='Profile picture of Robbie Robot.' />}
```

Comment:
empty alt text should suffice because the user name is already exposed
```suggestion
        image={<img src={imgSrc} alt='' />}
```

same comment in other examples

Reply:
Done!